src/init.cc

```diff
@@ -136,6 +136,83 @@ static void initOnceFunc() {
 exit:;
 }
 
+std::string trimString(const std::string& s) {
```

Comment:
Better move these new functions to https://github.com/ROCm/rccl/blob/develop/src/rccl_wrap.cc